pygmt/datasets/samples.py

```diff
@@ -131,36 +125,32 @@ def _load_hotspots():
     )
 
 
-def _load_mars_shape():
+def _load_mars_shape() -> pd.DataFrame:
     """
-    Load a table of data for the shape of Mars as a pandas.DataFrame.
+    Load a table of data for the shape of Mars.
 
-    Data and information are from Smith, D. E., and M. T. Zuber (1996),
-    The shape of Mars and the topographic signature of the hemispheric
-    dichotomy.
+    Data and information are from Smith, D. E., and M. T. Zuber (1996), The shape of
+    Mars and the topographic signature of the hemispheric dichotomy.
 
     Returns
     -------
-    data : pandas.DataFrame
-        The data table with column names "longitude", "latitude", and
-        "radius_m".
+    data
+        The data table with column names "longitude", "latitude", and "radius_m".
```

Comment:
To make the formulations consistent.

```suggestion
        The data table. The column names are "longitude", "latitude", and "radius_m".
```